roles/install-etcd/tasks/main.yml

```diff
@@ -0,0 +1,36 @@
+- name: Download and setup etcd
+  unarchive:
+    src: "https://storage.googleapis.com/etcd/{{ etcd_version }}/etcd-{{ etcd_version }}-linux-ppc64le.tar.gz"
+    dest: "/usr/local/bin"
+    remote_src: yes
+    extra_opts:
+      - --strip-components=1
+  retries: 3
+  delay: 5
+
+- name: Generate etcd.service
+  template:
```

Comment:
Just a thought, if we can rather performing the `copy` OP than `template`, as long as we're not going to plan on re-using variables in the .j2 file. 

I agree, template can get the same job done. 😃 

Ref: https://github.com/ppc64le-cloud/k8s-ansible/blob/520f1241bef84304e3f1e8cc320d2ceb1e4f4774/roles/runtime/tasks/crio.yaml#L46-L51C1

Reply:
Done